workflow-templates/im-test-k6-ci.properties.json

```
@@ -0,0 +1,5 @@
+{
+    "name": "Test - K6 Workflow for daily CI",
+    "description": "Workflow template for running K6 performance tests with every merge to master",
```

Sorry I missed this one before.
```suggestion
    "description": "Workflow template for running K6 performance tests with every merge to main",
```

---

I think this one does need to change, since we're running it for the main branch.